enhancements/network-observability/network-observability.md

```diff
@@ -161,6 +174,100 @@ prior to enabling network observability, will require additional resources.
 The specific number of cores and the amount of memory and storage required
 will be finalized as more testing is done.
 
+### CRD
+
+The operator custom resource definition (CRD) is divided per components whith 
+the ability to select or enable optionnal ones. The console dynamic plugin also 
+have an option to automatically register.
+
+```yaml
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: "network-observability"
+  agent: #either ipfix or ebpf
+  ipfix:
+    ...
+  ebpf:
+    ...
+  flowlogsPipeline:
+    ...
+  kafka:
+    enable: false
+    ...
+  loki:
+    ...
+  consolePlugin:
+    register: true
+    ...
+```
+
+Here is a full sample using `ipfix` (current default configuration):
```

Comment:
Maybe better to put a link to this file instead of the actual sample here to not clutter the document.  Also if the default configuration changes, it won't get out-of-date.